### ISSUE TITLE
Use a custom dataset for training

### ISSUE THREAD

**Issue by AsishKotaru550** [ChenHongruixuan/MambaCD#51]
I want to use a custom dataset for training the model. This dataset images contain 9 input channels and the labels are just 0s and 1s. How do I go about modifying the model parameters....
For starters, I changed the number of input channels in configs/config.py of MambaBCD; but the model has many components and I might have missed changing few parameters.

**Comment by ChenHongruixuan** [comment]
Hi,

Thank you for your question. 

We also tested our ChangeMamba model on OSCD dataset. The input under this dataset is 13 bands instead of three, matching your needs. All you need to do is add the following code in the training script: 

> self.deep_model = STMambaBCD(...) # Model definition 
> self.deep_model.encoder.patch_embed[0] = nn.Conv2d(13, 48, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1)) # modify the first convolutional layer
> ...

Best,
Hongruixuan

**Comment by AsishKotaru550** [comment]
Changed `in_chans` in config.py which took care of it.
Thanks.
